### DISCUSSION TITLE
next/link `scroll={false}` for hash urls not working as expected

### DISCUSSION THREAD

**Discussion by saevarb** [vercel/next.js#46604] [Help]
I just found this issue(https://github.com/vercel/next.js/issues/46456)  after having issues with `scroll={false}` on next's `Link` not having any effect. I.e. we're trying to navigate between sections on the same site(ideally with smooth-scroll behavior) but no matter what we do, next is scrolling to the top of the page before then jumping down to the section, resulting in a very janky experience.

Should I understand this as hash routing simply being broken right now and being on the roadmap? If so, where is this documented?

**Comment by TSIA-SN** [comment]
I'm also having the same issue on my page, switching to `<a>` tags fixes it, but obviously that's not ideal.

For additional context, the site is using the pages directory and I'm using NextJS 13.4.17

**Comment by don-esteban** [comment]
I remember I had issues too some month ago during evaluation. Today we use latest 14 version and I just confirmed that fragment links are doing well. Plain `<Link>` works fine.
